src/Subscriber/RewriteUnwantedPhrasesSubscriber.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Api\PullRequest\PullRequestApi;
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Lock\LockFactory;
+
+/**
+ * There are some phrases that should be avoided in title/description. This
+ * subscriber helps rewrite those.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class RewriteUnwantedPhrasesSubscriber implements EventSubscriberInterface
+{
+    private $pullRequestApi;
+    private $lockFactory;
+
+    public function __construct(PullRequestApi $pullRequestApi, LockFactory $lockFactory)
+    {
+        $this->pullRequestApi = $pullRequestApi;
+        $this->lockFactory = $lockFactory;
+    }
+
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        $action = $data['action'];
+        if (!in_array($action, ['opened', 'ready_for_review', 'edited'])) {
```

Comment:
This type of check needs to be centralized. We've got `ready_for_review`, for example, in 7 different subscribers now :) 

Perhaps we could introduce a "model" class that we pass the `GitHubEvent` into (e.g. `new GithubPullRequestData($event)`) and then we can call things like `$pullRequest->isInOpenState())` or something similar.

Can be done in another PR.